aqua/qt4-mac/Portfile

```diff
@@ -437,7 +449,7 @@ post-patch {
     # were all patched in (2) above, and can be easily changed or
     # overridden by the user in a project-local qmake .pro script.
 
-    set TARGET "10.${MINOR}"
+    set TARGET "${MAJOR}.${MINOR}"
```

Comment:
Why is the port deciding what deployment target to use, rather than using the value MacPorts asked it to use?

Reply:
It is, just a bit indirectly, as

```
    set MAJOR [lindex [split ${macosx_deployment_target} "."] 0]
    set MINOR [lindex [split ${macosx_deployment_target} "."] 1]
```

I agree though it would be better to just use the target directly here.